pipelines/pipeline_katello_nightly.yml

```diff
@@ -11,35 +11,42 @@
   roles:
     - forklift
 
-- hosts: pipeline-katello-nightly-centos7
+- hosts:
+  - pipeline-katello-nightly-centos7
+  - pipeline-proxy-nightly-centos7
   become: yes
   vars:
     katello_repositories_version: nightly
     foreman_repositories_version: nightly
     katello_repositories_use_koji: true
+    foreman_installer_skip_installer: true
+  roles:
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - update_os_packages
+    - foreman_installer
```

Comment:
I was thinking of extending the tmp boxes to also set variables. Then you can set it at the box level and the foreman_installer role could install both. I'm going to experiment with this when I have time.